editor/components/block-list/block.js

```diff
@@ -387,18 +387,20 @@ export class BlockListBlock extends Component {
 			const cloneWrapper = document.createElement( 'div' );
 			const clone = block.cloneNode( true );
 			const blockRect = block.getBoundingClientRect();
+    		const blockTopOffset = parseInt( blockRect.top, 10 );
```

Comment:
Indentation here seems a bit off.

Reply:
Thanks - spotted a few other linting errors too, but should be cleared in latest commit. Just run the linter and all green. 😃

Reply:
👍